oximeter/collector/tests/output/self-stat-schema.json

```diff
@@ -39,7 +39,7 @@
       }
     ],
     "datum_type": "cumulative_u64",
-    "created": "2024-05-03T22:37:51.326086935Z"
+    "created": "2024-05-11T09:41:23.361298682Z"
```

Comment:
Was this supposed to be committed?

Reply:
Hmm, it changed. But maybe I shouldn't have committed it?

Reply:
I think I would vote "don't commit it" along with possibly file an issue? I'm surprised this isn't an issue on main...

Reply:
This works because we explicitly ignore timestamps when comparing the schema on disk with the current one. It's fine to commit, or not, whatever you'd like.

Reply:
That makes sense. Thanks @bnaecker. I believe it changed when I ran all the tests with `cargo nextest run`. I'll leave it out of sheer laziness.